libraries/AP_BattMonitor/AP_BattMonitor_Backend.cpp

```diff
@@ -161,13 +161,21 @@ bool AP_BattMonitor_Backend::arming_checks(char * buffer, size_t buflen) const
                                 (_state.voltage < _params._arming_minimum_voltage);
     bool below_arming_capacity = (_params._arming_minimum_capacity > 0) &&
                                  ((_params._pack_capacity - _state.consumed_mah) < _params._arming_minimum_capacity);
+    bool fs_capacity_inversion = is_positive(_params._critical_capacity) &&
+                                 is_positive(_params._low_capacity) &&
+                                 (_params._low_capacity < _params._critical_capacity);
```

Comment:
Should this really be `<=`?  I expect the handling code is fine, but having them equal seems also to be a misconfiguraiton.

Reply:
Hmm... Probably is, I was less worried about it, but probably worth forcing correctness here :)